.github/workflows/add_issue_to_project.yml

```diff
@@ -0,0 +1,20 @@
+name: Add new issue/PR to project
+
+on:
+  issues:
+    types:
+      - opened
+      
+  pull_request:
+    types:
+      - opened
+
+jobs:
+  add-to-project:
+    name: Add issue or PR to project
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/add-to-project@v0.3.0
+        with:
+          project-url: https://github.com/orgs/rapidsai/projects/41
+          github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I've added a secret called `ADD_TO_PROJECT_GITHUB_TOKEN` for this purpose. Kindly update the name in the workflow and you should be good to go.

Reply:
Done ty!